### DISCUSSION TITLE
Can I import multiple files into one project?

### DISCUSSION THREAD

**Discussion by SagevdBrand** [asreview/asreview#574] [Q&A] [answered]
Is it possible to upload multiple files from different search engines into a single project?

**Comment by boer0107-zz** [comment, marked as answer]
We advise you to use a reference manager to import results from multiple databases, deduplicate these items and export the titles into one single file (.ris in most reference managers, some like Zotero also support csv-export). ASReview does not support multiple imports at this moment.
ASReview compatibility for single files: https://asreview.readthedocs.io/en/latest/data.html#compatibility
